### DISCUSSION TITLE
Fault SO2.3.100 Distributed Deployment

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#7407] [Unsupported Versions] [answered]
Version 2.3.100 Distributed Deployment

Currently receiving fault errors on all Search and Forward nodes. Data seems to be coming in just fine. How can we diagnose what is causing the fault error? Is there a log file we can reference? There is no firewall in place. 

![image](https://user-images.githubusercontent.com/70167373/156415618-88e0e47c-3d58-4a48-abc1-e6179bca02c5.png)

**Comment by dougburks** [comment, marked as answer]
Have you tried running `so-status` on the nodes?

**Reply by ColeVan** [reply]
Correct. No issues on the nodes themselves. After adjusting the NTP setting in chrony.conf it seemed to clear up. Still testing....

**Reply by dougburks** [reply]
Yes, time synchronization is important.

**Reply by ColeVan** [reply]
Question? Is there a log file we can reference for fault errors so we can diagnose faults faster?

**Reply by dougburks** [reply]
I've added the following to https://docs.securityonion.net/en/2.3/grid.html:
> If any nodes show anything other than OK, then run [so-status](https://docs.securityonion.net/en/2.3/so-status.html#so-status) on the node and check logs in /opt/so/log/.

**Reply by ColeVan** [reply]
Awesome! Thanks for all your help!
